tools/bin/commands/release

```diff
@@ -262,6 +272,9 @@ create_upgrade_docker_image() {
     # Copy syndesis.yml
     cp $topdir/install/syndesis.yml .
 
+    # Copy over syndesis-cli jar
+    cp $topdir/app/server/cli/target/syndesis-cli.jar .
+
     # Create the image
     docker build -t syndesis/syndesis-upgrade:${release_version} --build-arg version=${release_version} .
```

Comment:
Would it be better if the `cp` above is a part of the syndesis-upgrade Dockerfile?

Reply:
You can't add files in a Dockerfile which is outside the context directory. I.e. it must be in the same directory or below the Dockerfile.

Therefore we have to copy over it to the Dockerfile's dir.

Reply:
Same in the `build.sh`.

But shouldn't be a problem as the syndesis-cli created here during the release process it guaranteed to be created fresh and fitting to the version we are building.